src/test/java/loanbook/testutil/LoanBookForRemindTest.java

```diff
@@ -0,0 +1,31 @@
+package loanbook.testutil;
```

Comment:
Should we put this code together with the TypicalLoanBook java file? They are both factory methods, and are representations of typically what happens in the LoanBook.

Reply:
Fixed. Thank you so much!